pyproject.toml

```diff
@@ -122,13 +121,16 @@ split-on-trailing-comma = false
     "D",  # we don't need public-API-polished docstrings in tests.
     "FBT",  # using a boolean as a test object is useful!
     "PLR",  # likewise using specific numbers and strings in tests.
+    "A",
```

Comment:
newer versions of ruff include more rules, which were triggering on tests.  This ignores them.